ccd

```diff
@@ -76,6 +82,7 @@ function tag_set() {
     touch $TAGS_FILE
     sed -i '' "/$tagEnv/d" $TAGS_FILE
     echo "export $tagEnv=$tagName" >> $TAGS_FILE
+    echo "export ${tagEnv}_LOCAL=''" >> $TAGS_FILE
```

Comment:
just to make the compose file a bit more readable, any chance we could call these env vars `CCD_API_GATEWAY_REPOSITORY_LOCAL` rather than `CCD_API_GATEWAY_TAG_LOCAL`?
Only if that would be easy to do. 
Is it possible for example on this line to replace with sed the _TAG from the ${tagEnv} with _REPOSITORY?

Reply:
Yeah wondered myself tbh as its unreadable naming.  Was hoping you'd just ignore it :)  Will figure out some fix.  I don't really like the name I've got anyway might go with ${REPO_NAME}_USE_LOCAL since it is a boolean in some way